prometheus/templates/app_alert.rules.tmpl

```diff
@@ -0,0 +1,96 @@
+groups:
+- name: High CPU
+  rules:
+%{ for instance in apps ~}
+  - alert: High-CPU-${instance.app_name}
+    expr: avg by ( app ) (cpu{app=~"${instance.app_name}"}) > ${instance.max_cpu}
+    for: 5m
+    annotations:
+      summary:     ${instance.app_name} High CPU Alert
+      dashboard:   ${cfapps_dashboard_url}&var-Applications=${instance.app_name}
+      description: "CPU usage has increased in the last 5 minutes (current value: {{ $value }}%)"
+    labels:
+      environment: ${instance.monitoring_space_name}
```

Comment:
This is not an environment but a space, we can remove this label

Reply:
removed label.